middleware/assignLabels.js

```diff
@@ -0,0 +1,23 @@
+function setup(labelGenerator) {
```

Comment:
I would prefer setting the default in here to be `pelias-labels` and only override if another label generator has been passed in for testing or whatever. In my opinion, that makes the client code cleaner.